b2/sync.py

```diff
@@ -545,9 +545,10 @@ def _walk_relative_paths(self, prefix_len, dir_path, reporter):
                 )
             full_path = os.path.join(dir_path, name)
             relative_path = full_path[prefix_len:]
+            # Skip broken symlinks or other inaccessible files
             if not os.path.exists(full_path):
-                # Skip broken symlinks or other inaccessible files
-                reporter.local_access_error(full_path)
+                if reporter:
```

Comment:
I generally do (thank you for noticing :)) , but checking against `None` is a special case. If one day the SyncReporter class renamed to SyncStatus would implement casting to boolean as a way to check if it contains any errors (something that I think could actually happen), we wouldn't like the `if` in question to suddenly start filtering input about bad symlinks after the first one. That would create a bug which is likely to slip through the test harness and would be rare, possibly irreproducible and hard to debug once reported.